includes/class-sensei-settings-api.php

```diff
@@ -1033,6 +1033,7 @@ public function enqueue_styles() {
 
 		wp_enqueue_style( 'farbtastic' );
 		Sensei()->assets->enqueue( 'sensei-settings-api', 'css/settings.css', [ 'farbtastic' ] );
+		Sensei()->assets->enqueue( 'sensei-settings', 'css/sensei-banner.css' );
```

Comment:
I'm wondering if we should just put all the banner CSS in the existing `settings.css`? Both files are loaded anyway, and to me it seems a bit cleaner to have one file with all the CSS needed for a particular screen, and add comments in the file to break up the different sections.

Reply:
Soo, I didn't do it like that originally since I wasn't sure if we are going to have banner on some other place also. But I've returned it to settings.scss

https://github.com/Automattic/sensei/pull/4801/files#diff-5c437b51ca7067e0a3307a080cfa9c2315d7fe7d62744683bd4972bbd62869deR539